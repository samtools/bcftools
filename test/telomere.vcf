##fileformat=VCFv4.2
##contig=<ID=22>
#CHROM	POS	ID	REF	ALT	QUAL	FILTER	INFO
22	0	id0	C	G	.	.	.
22	1	id1	C	G	.	.	.
