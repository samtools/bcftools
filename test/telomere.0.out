22	0	id0	C	G	.	.	.
